app/src/main/java/com/nexters/fullstack/ui/activity/LabelOutAppActivity.kt

```diff
@@ -0,0 +1,52 @@
+package com.nexters.fullstack.ui.activity
+
+import android.content.Intent
+import android.net.Uri
+import android.os.Bundle
+import android.os.Parcelable
+import androidx.fragment.app.Fragment
+import com.nexters.fullstack.R
+import com.nexters.fullstack.base.BaseActivity
+import com.nexters.fullstack.databinding.ActivityLabelOutappBinding
+import com.nexters.fullstack.ext.loadFragment
+import com.nexters.fullstack.ui.fragment.LabelOutAppFragment
+import com.nexters.fullstack.viewmodel.LabelOutAppViewModel
+import org.koin.androidx.viewmodel.ext.android.viewModel
+
+class LabelOutAppActivity : BaseActivity<ActivityLabelOutappBinding, LabelOutAppViewModel>() {
+    override val layoutRes: Int = R.layout.activity_label_outapp
+    override val viewModel: LabelOutAppViewModel by viewModel()
+    private val labelOutAppFragment = LabelOutAppFragment.getInstance()
+    private lateinit var activeFragment: Fragment
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        when{
+            intent?.action == Intent.ACTION_SEND
+                && intent.type?.startsWith("image/") == true -> {
```

Comment:
"image/" 같은건 변수로 빼는게 어떨까 ?

```suggestion
intent.type?.startsWith(IMAGE_PRIFIX) == true -> {
...

companion object {
    private const val IMAGE_PRIFIX = "image/"
}
```